docs/DeveloperGuide.md

```diff
@@ -378,234 +375,353 @@ Use case ends.
 
 Use case ends.
 
 **Extensions**
 
-* 3a. SellSavvy detects required details missing.
-    * 3a1. SellSavvy displays an error message “Invalid command format!” and states the command format.
+* 2a. SellSavvy detects there are required parameters missing or the customer index is non-positive.
+    * 2a1. SellSavvy displays an error about the format and states the command format.
 
   Use case ends.
 
 
-* 3b. SellSavvy detects that there is a parameter not satisfying its constraint.
-    * 3b1. SellSavvy states the constraint of the invalid parameter.
+* 2b. SellSavvy detects that there is a parameter not satisfying its constraints.
+    * 2b1. SellSavvy states the constraints of the invalid parameter.
 
   Use case ends
 
 
-* 3c. SellSavvy detects that there are no customers with the specified index.
-    * 3c1. SellSavvy displays an error that the customer index is invalid.
+* 2c. SellSavvy detects that there are no customers with the specified index.
+    * 2c1. SellSavvy displays an error that the customer index is invalid.
 
   Use case ends.
 
 
-* 4a. SellSavvy detects that there is an existing pending order under the customer with a similar details.
-    * 4a1. SellSavvy gives a warning that an order with a similar details already exists.
+* 3a. SellSavvy detects that there is an existing pending order under the customer with a similar details.
+    * 3a1. SellSavvy gives a warning that an order with a similar details already exists.
 
-  Use case resumes from step 5.
+  Use case resumes from step 4.
 
-**Use case 6: List a Customer's Orders**
+**Use case 7: List a Customer's Orders**
 
-* **System**: SellSavvy
-* **Actor**: User
-* **Use Case**: UC06 - List a Customer's Orders
-* **Preconditions**: There are customers displayed in the customer list.
+* **Use Case**: UC07 - List a Customer's Orders
+* **Preconditions**: There are orders displayed in the customer's list.
 * **Guarantees**:
     * Orders made by specific customer will be displayed as a list, if input parameters are valid.
 
 **MSS**
 
-1. User wants to view all orders made by a specific customer.
-2. User finds the customer index from the customer list.
-3. User inputs command to list all orders, by the index of customer in customer list.
-4. SellSavvy retrieves a list of all orders made by specified customer.
-5. SellSavvy displays the orders in a list in GUI.
+1. User finds the index of the customer in the customer list whose orders they want to view.
+2. User inputs command to list all orders under the index of customer.
+3. SellSavvy displays the orders under the specified customer.
 
 Use case ends.
 
 **Extensions**
 
-* 3a. SellSavvy detects that there are no customers with the specified index.
-    * 3a1. SellSavvy displays an error that the customer index is invalid.
+* 2a. SellSavvy detects the customer index is missing or non-positive.
+    * 2a1. SellSavvy displays an error about the format and states the command format.
+
+  Use case ends.
+
```

Comment:
what about the no related orders found? I think someone reported it for find this works the same way?

Reply:
will add